releasenotes/notes/stream-ep-cli-command-ab47ad6c770ffa57.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Adds a new agent command `stream-ep` to stream the event platform payloads being generated by the agent.
```

RST uses double backticks for monospace text.

```suggestion
    Adds a new agent command ``stream-ep`` to stream the event platform payloads being generated by the agent.
```